README.md

```diff
@@ -2,22 +2,50 @@
 
 Collection of artifacts to test SonataFlow Use Cases TP2.
 
-## Deploy Data Index locally
-
-Example of how to deploy Data Index on Kubernetes.
-
-### Prereqs
+## Prereqs for all the use cases
 
 1. Minikube installed
 2. kubectl installed
+3. SonataFlow operator installed if workflows are deployed
+
```

Comment:
Add the following: 

The minikube registry addon must be enabled in order for the workflow images to get stored in the local registry of the container.
```
minikube addons enable registry 
```
To confirm that the add is enabled:
```
minikube addons list |grep registry
| registry                    | minikube | enabled ✅   | minikube                       |
| registry-aliases            | minikube | disabled     | 3rd party (unknown)            |
| registry-creds              | minikube | disabled     | 3rd party (UPMC Enterprises)   |
```

Reply:
@jianrongzhang89 do you mind also sending a PR here: https://sonataflow.org/serverlessworkflow/latest/cloud/operator/build-and-deploy-workflows.html#building-kubernetes

This file: https://github.com/asf-transfer/kogito-docs/blob/main/serverlessworkflow/modules/ROOT/pages/cloud/operator/build-and-deploy-workflows.adoc

We are in a transition to apache, we might need to wait a little bit to stabilize. By tomorrow we should be fine.

Reply:
@ricardozanini sounds good. I will submit a PR for it.